code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
No, the prompts can get excessive, it should default to secret and print some line in the chatbox with a link to mark it public